### PR TITLE
chore: use includes instead indexOf

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [x] Other (about what?)

### 💡 Background and solution

WX20221021-103912

WX20221021-104003

`includes`用的是 `SameValueZero` 判断方式，而 `indexOf` 用的是 `IsStrictlyEqual` 判断方式

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | chore: use includes instead indexOf |
| 🇨🇳 Chinese | chore: use includes instead indexOf |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by yoyo837** [comment]
我咋记得有些地方 是 indexOf 优于 includes? 

https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-includes.md

**Comment by li-jia-nan** [comment]
> 我咋记得有些地方 是 indexOf 优于 includes?
> 
> https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-includes.md

可以举个例子吗大佬？

**Comment by li-jia-nan** [comment]
> 我咋记得有些地方 是 indexOf 优于 includes?
> 
> https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/prefer-includes.md

我看到你发的这个链接里面好像是优先使用 `includes` 的？

File: scripts/check-commit.js [hunk 1]
```diff
@@ -44,7 +44,7 @@ async function checkCommit({ files }) {
 
 async function checkRemote() {
   const { remote } = await git.fetch('origin', 'master');
-  if (remote?.indexOf('ant-design/ant-design') === -1) {
+  if (!remote?.includes('ant-design/ant-design')) {
```
Comment (MadCcc) [inline scripts/check-commit.js:47]: 这种逻辑不对了

Comment (li-jia-nan) [reply scripts/check-commit.js]: 想了一下，remote 的值为 null 的时候确实不对了，改回去了

**Review by MadCcc — APPROVED** [review]

**Pull request merged**